django_ai_assistant/helpers/assistants.py

```diff
@@ -464,15 +388,23 @@ def as_graph(self, thread_id: Any | None = None) -> Runnable[dict, dict]:
         Returns:
             the compiled graph
         """
+        # DjangoChatMessageHistory must be here because Django may not be loaded yet elsewhere.
+        # DjangoChatMessageHistory was used in the context of langchain, now that we are using
+        # langgraph this can be further simplified by just porting the add_messages logic.
```

Comment:
Is this tracked? So we don't need the DjangoChatMessageHistory wrapper anymore?

Reply:
Yes, it seems like it could be replaced by one simple method that retrieves the message history and another that adds new ones. I created an issue for this https://github.com/vintasoftware/django-ai-assistant/issues/169